projects/packages/assets/src/class-assets.php

```diff
@@ -353,6 +355,15 @@ function ( $d ) {
 		if ( $options['async'] ) {
 			self::instance()->add_async_script( $handle );
 		}
+		if ( $options['textdomain'] ) {
+			wp_set_script_translations( $handle, $options['textdomain'] );
+		} elseif ( in_array( 'wp-i18n', $options['dependencies'], true ) ) {
+			_doing_it_wrong(
+				__METHOD__,
+				/* translators: %s is the script handle. */
+				esc_html( sprintf( __( 'Script "%s" depends on wp-i18n but does not specify "textdomain"', 'jetpack' ), $handle ) )
+			);
```

Comment:
Hi @anomiex.
The function seems to be missing the third argument, throws a fatal.

Reply:
Huh. I thought I had seen code elsewhere leaving that parameter out. Should have paid more attention.

#21852